src/main/java/frc/robot/shooter/commands/SpinUp.java

```diff
@@ -9,17 +9,26 @@
 public class SpinUp extends Command {
   private final ShooterSubsystem subsystem;
   private final BooleanSupplier slowShoot;
+  private final double targetSpeed;
 
-  public SpinUp(ShooterSubsystem subsystem, BooleanSupplier slowShoot) {
+  public SpinUp(ShooterSubsystem subsystem, double targetSpeed, BooleanSupplier slowShoot) {
     this.subsystem = subsystem;
     this.slowShoot = slowShoot;
+    this.targetSpeed = targetSpeed;
+    addRequirements(subsystem);
+  }
+
+  public SpinUp(ShooterSubsystem subsystem, double targetSpeed) {
+    this.subsystem = subsystem;
+    this.slowShoot = () -> false;
+    this.targetSpeed = targetSpeed;
     addRequirements(subsystem);
   }
 
   @Override
   public void initialize() {
     double scale = slowShoot.getAsBoolean() ? Constants.Shooter.AMP_SHOOT_SCALE : 1;
-    subsystem.setTargetShooterSpeed(Constants.Shooter.SHOOT_SPEED * scale);
```

Comment:
This is why i think you need to switch to separate targets, if you want the close shot to remain different. Right now the long shot would shoot at 8000 and 7200.

https://github.com/frc1675/frc1675-2024/blob/main/src/main/java/frc/robot/shooter/ShooterSubsystem.java#L63

Reply:
This comment was supposed to be on the new version of this

Reply:
I'm confused? are you referring to the wheels running at different speeds?